src/commands/ginkgo-v2-labels.yml

```
@@ -21,8 +21,7 @@ parameters:
   golang_version:
     description: >
       NOTE these are installed by https://github.com/myhelix/circleci-runner/blob/master/lib/circleci-runner-stack.ts#L152..L156
-    type: enum
-    enum: ['1.18', '1.19', '1.20', '1.21', '1.22', '1.23', '1.24']
+    type: string
```

is there a way to validate with a regex or something, I think it would be useful to fail fast. Otherwise looks good